packages/api/CHANGELOG.md

```diff
@@ -4,6 +4,20 @@ All notable changes to this project will be documented in this file.
 See [Conventional Commits](https://conventionalcommits.org) for commit guidelines.
 
 ## [UNRELEASED]
+## [1.3.0-alpha.1](https://github.com/cennznet/api.js/compare/release/1.2.1...prerelease/1.3.0-alpha.1) (23/10/2020)
+### Added:
+    - bump http-proxy from 1.18.0 to 1.18.1
+    - Add the publish CircleCi job for the release branches
+### Changed:
+    - Updated the code base to use the current latest polkadot dependencies (^2.3.1 as of now) 
```

Comment:
is it easy/hard to pull out some of the breaking changes from the polkadot changelog to put here?

Reply:
will check..